### ISSUE TITLE
New brain images! 

### ISSUE THREAD

**Issue by jonmigueljara** [berkeley-stat159/project-eta#85]
Hey Matt. We updated our design matrix so that we are convolving the gain and loss regressors as well like you suggested.

![screen shot 2015-12-12 at 3 47 04 am](https://cloud.githubusercontent.com/assets/8962834/11761747/be75db80-a083-11e5-9266-e44d05bc2828.png)

![screen shot 2015-12-12 at 3 45 04 am](https://cloud.githubusercontent.com/assets/8962834/11761752/dcfc4530-a083-11e5-9b41-846070514b6b.png)

Can you help us interpeting these images? We notice that there are some small clusters of activiation, but what else can we do to analyze and interpet these images?

**Comment by matthew-brett** [comment]
Are these images from analysis of the smoothed data?  Preprocessed data?   What else is in the design matrix?  Could you also post an image of the design matrix?

**Comment by jonmigueljara** [comment]
yeah this is the data we smoothed but its not the preprocessed data that we got towards the end of the class. The design matrix has the convolved gain,loss, and onset values. 

These are the values.
![design_plot_task_onset](https://cloud.githubusercontent.com/assets/8962834/11763963/511db6d4-a0d3-11e5-9487-5456141802cd.jpg)

![design_plot_task_onset](https://cloud.githubusercontent.com/assets/8962834/11763965/5bf9efb4-a0d3-11e5-84da-dbef356445e3.jpg)

**Comment by matthew-brett** [comment]
What other regressors did you include?

Please post an 'imshow' of your design matrix. 

What smoothing did you use?

**Comment by jonmigueljara** [comment]
the 3 regressors are. the general task onset times, gain, and loss. All 3 convolved.

we smoothed using a Gaussian Filter by 2 SD's in all 3 spatial dimensions.

oh right, here it is:

![screen shot 2015-12-12 at 1 40 42 pm](https://cloud.githubusercontent.com/assets/8962834/11764045/3e9d6038-a0d6-11e5-8b8d-862796c3576b.png)

**Comment by matthew-brett** [comment]
Is the first column a column of 1?

Did you try including the linear drift term?  The squared linear drift term?  The PCA components that looked artefactual?   The reason I ask is because your image looks noisy and seems to be picking up the edge of the brain, so might be picking up some movement effects.

I also recommend you subtract the mean from your convolved regressors.   This makes the betas easier to interpret, because you can interpret the 'gain' and 'loss' betas as the increase / decrease in activity having allowed for activity due to the task event (your 'onset' regressor).

**Comment by jonmigueljara** [comment]
yes the first column is 1's for the intercept. We were not including the drift terms so I attempted to use them but I found that it still looks "edgey". Is this still possible due to noise?

![image](https://cloud.githubusercontent.com/assets/8962834/11766260/5057931c-a134-11e5-9373-3018dea7d6f5.png)

**Comment by matthew-brett** [comment]
I agree - it does look very edgey.  I would try subtracting the mean from the regressors, and I would try also looking at the PCA of the data to see if you can find edgey components to remove by regression.
